README.md

```diff
@@ -1,27 +1,87 @@
 
+### Submission deadlines
+Submissions are due at 8pm ET every Wednesday. This time was chosen to give modelers time in the beginning of the week to run and adjust models and stakeholders time at the end of the week to incorporate preliminary results into discussions or decision making.
 
-If you have questions aboutthis target, please reach out to *[insert name]*  (*[insert email]* ). 
+### Processed datasets, selecting variant clades
+Early Monday morning (~3am ET) prior to a Wednesday on which submissions are due, the hub generates a plain text file whose sole contents are a list of  [NextClade clade names](https://clades.nextstrain.org/about) that will be accepted in submission files for the upcoming deadline. The plain text file will live in the `auxiliary-data/modeled-clades/` directory of the repository and will be named "YYYY-MM-DD.txt" where "YYYY-MM-DD" is the date of the Wednesday on which submissions are due.
 
-## Acknowledgments
-This repository follows the guidelines and standards outlined by the [hubverse]([url](https://hubdocs.readthedocs.io/en/latest/)), which provides a set of data formats and open source tools for modeling hubs.
+<!-- TODO: consider revising based on Spencer's concern.-->
+Each week the hub designates up to nine NextStrain clades with the highest reported prevalence of at least 1% across the US in any of the three complete MMWR weeks preceding the Wednesday submission date. Any clades with prevalence of less than 1% are grouped into an “other” category for which predictions of combined prevalence are also collected. No more than 10 clades (including “other”) are selected in a given week.
 
+This clade selection is based on the [NextStrain USA sequence count file based on US data from GenBank](https://data.nextstrain.org/files/workflows/forecasts-ncov/open/nextstrain_clades/usa.tsv.gz), using [this script](https://github.com/reichlab/virus-clade-utils/blob/main/src/virus_clade_utils/get_clade_list.py). The NextStrain file with counts is [typically updated daily in the late evening US eastern time](https://github.com/nextstrain/forecasts-ncov/actions/workflows/update-ncov-open-clade-counts.yaml) (it is only updated when new data are available). The hub pulls the most recent version of the file when the workflow runs each week. The precise lineage assignment model (sometimes referred to as a “reference tree”) that was used as well as the version of raw sequence data is be stored as metadata, to facilitate reproducibility and evaluation. 
+<!-- TODO: Check this -->Versions of these target data files are maintained by the hub, so modeling teams can come back and retrospectively test their models on data as they were available in real time.
```

Comment:
I'm not quite sure what we want to save here: the sequence metadata we used to generate the clade list? The reference trees? Both?

@elray1 and I discussed saving the reference trees, and I created two related issues:
- https://github.com/reichlab/virus-clade-utils/issues/20
- https://github.com/reichlab/variant-nowcast-hub/issues/41

Happy to chat synchronously about what we want to do here, if that's easier!